notebooks/proj_sys_eng/sitcom_integration/LVV-T2667-ComCam_Configuration_Events_Verification.ipynb

```diff
@@ -0,0 +1,3665 @@
+   },
+   "source": [
+    "# Test Case LVV-T2667\n",
+    "\n",
+    "The purpose of this test case is to verify the ComCam Events related to the configuration are published to the EFD as defined in XMLv12.0.0.\n",
+    "Check that the values from the EFD are consistent with the values from the configuration file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "477e80a8-3345-40c9-b241-bb00d3c1fdcc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Executed by ksiruno-lsst on 2022-08-19T16:12:16.441.\n",
+      "  Running in pillan04 at tucson\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from lsst.sitcom import vandv\n",
+    "\n",
+    "exec_info = vandv.ExecutionInfo()\n",
+    "print(exec_info)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee6d5fc1-8114-4514-bed4-ff15dc061841",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "cc3a4c40-5c5d-45ff-8729-d312582f55fc",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import asyncio\n",
+    "import logging\n",
+    "import os\n",
+    "import yaml\n",
+    "\n",
+    "import astropy.units as u\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from astropy.time import Time\n",
+    "from datetime import datetime, timedelta\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "from lsst.ts.observatory.control.maintel import ComCam\n",
+    "from lsst_efd_client import EfdClient\n",
+    "from lsst.ts import salobj"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a18d235b-2221-4e7a-aa50-2a34072ba3c1",
+   "metadata": {},
+   "source": [
+    "The following block sets the necessary environment variables for setting up the DDS/SAL communication"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "9aed0184-06b7-463f-8a43-00558cded4d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ[\"LSST_DDS_HISTORYSYNC\"] = \"30\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "acd6c870-7fe8-408e-b00c-41915653622a",
+   "metadata": {},
+   "source": [
+    "Setting up logger"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "c468edd4-e263-42fe-b50a-97629c61d887",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logger = logging.getLogger(\"LVV-T2267\")\n",
+    "logger.setLevel(logging.DEBUG)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "373219ab-4c68-4969-af53-421eb62e356d",
+   "metadata": {},
+   "source": [
+    "Getting unique index for Script"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "2d91837c-889f-443d-8349-93d71f4fd391",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logger.info(f'Your UID is {os.getuid()}')\n",
```

Comment:
The `lsst.sitcom.vandv` module now has a new function that creates the index for the Script CSC automatically. You don't have to add it here, but I would recommend it. You can find it here: [`get_index`](https://github.com/lsst-sitcom/notebooks_vandv/blob/4f95f3237af9115ec435756a696c47f23e6e06d0/python/lsst/sitcom/vandv/info.py#L87).